myna/database/database_types.py

```diff
@@ -36,6 +37,14 @@ def return_datatype_class(datatype_str):
             return PeregrineHDF5(version=version)
         else:
             return PeregrineHDF5()
+    elif datatype_str.lower() in [
```

Comment:
Could split on "am", "bench", and "2022", with different separators instead

Reply:
added a helper function to clean the text string for database name input